### PR TITLE
add comments endpoint

### PR DESCRIPTION
**Pull request by jss118**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -143,3 +143,25 @@ describe("GET /api/articles", () => {
...
+    });
+  });
+  test("Status: 404 responds with an error for a valid but non-existing endpoint", async () => {
+    const { body } = await request(app)
+      .get("/api/articles/200/comments")
+      .expect(404);
+    expect(body.status).toBe(404);
+    expect(body.msg).toBe("article does not exist");
+  });
+});
```
Comment (HeatherEver) [inline __tests__/app.test.js:179]: couple of tests to add here - invalid id and a test for if the article exists but has no comments please :)

**Review by HeatherEver — COMMENTED** [review]
just a couple of changes please

File: __tests__/app.test.js (hunk 1 above)
Comment (jss118) [reply __tests__/app.test.js:179]: I've added the invalid id test, but shouldn't it be okay for an article to have no comments?

**Comment by cjplatten** [comment]
Looks good!

**Pull request merged**